projects/plugins/crm/tests/codeception/acceptance/11_JPCRM_Activation_Cest.php

```diff
@@ -26,22 +26,50 @@ public function catch_wp_update_pages( AcceptanceTester $I ) {
 		}
 	}
 
-	public function jpcrm_activation( AcceptanceTester $I ) {
-		// If it's installed, activate the plugin
+	public function test_single_plugin_activation_shows_wizard( AcceptanceTester $I ) {
 		$I->amOnPluginsPage();
 		$I->seePluginInstalled( 'jetpack-crm' );
-		$I->activatePlugin( 'jetpack-crm' );
 
-		// Activating the plugin directly loads the welcome wizard, so no need to move pages here.
+		// Click the activate link for jetpack-crm directly
+		$I->click( 'Activate', array( 'css' => 'tr[data-slug="jetpack-crm"] .activate a' ) );
 
-		// check no activation errors
+		// Check no activation errors and wizard is shown
 		$I->dontSeeElement( '#message.error' );
+		$this->assertWizardIsShown( $I );
+	}
+
+	public function test_bulk_plugin_activation_skips_wizard( AcceptanceTester $I ) {
```

Comment:
I suspect this doesn't show the wizard because technically the option already is set. So more accurately this is testing if the wizard shows on second load.